src/App.jsx

```diff
@@ -14,7 +15,7 @@ function App() {
       <Navbar />
       <Routes>
         <Route path="/" element={<Home />} />
-        {/* <Route index element={<BookList />} /> */}
+        <Route index element={<BookList />} />
```

Comment:
I can see you put a great efford to this project. However, If I clone your app and open it locally, it's not working as we expected. I can see only loading message:

<img width="762" alt="Screenshot 2023-06-30 at 19 36 16" src="https://github.com/Simpleshaikh1/Book-Store/assets/69011963/6d022306-0bec-40cf-8d0e-3e556fc15a13">

- [ ] This might cause by several reasons. I suggest you to check if your API has no issue and you are using correct api key. Or the error goes with loading method. The last one might be with dispatch. Kindly fix this issue. Due to this issue I was not able to review add and remove functionality of your app.